src/OmniSharp.Roslyn.CSharp/Services/Structure/MembersAsTreeService.cs

```diff
@@ -25,7 +25,8 @@ public async Task<FileMemberTree> Handle(MembersTreeRequest request)
         {
             return new FileMemberTree()
             {
-                TopLevelTypeDefinitions = await StructureComputer.Compute(_workspace.GetDocuments(request.FileName), _discovers)
+                // To provide complete members tree for the document wait until all projects are loaded.
+                TopLevelTypeDefinitions = await StructureComputer.Compute(await _workspace.GetDocumentsFromFullProjectModelAsync(request.FileName), _discovers)
```

Comment:
What happens if this just uses the single document overload?

Reply:
@rchande StructureComputer.Compute at the moment takes "IEnumerable<Document>" I'm assuming for a reason. In general though I was trying hard to make zero changes to the extension's behavior when on-demand-projects-loading is not enabled. Please advise if you think that only a single document can be used in either mode.

Reply:
Ok, let's leave this for me.